lib/emojiKeywords.json

```diff
@@ -83,5 +83,8 @@
   "react": "react",
   "apple": "appleinc",
   "cat": "cat",
-  "dog": "dog"
+  "dog": "dog",
+  "code": "male-technologist",
+  "hack": "male-technologist",
+  "autumn": "hackautumn"
```

Comment:
rn all of autumn posts are tagged with :aom: could we continue that trend?

Reply:
btw i created :aom: and i liked :hackautum: more than that :/

Reply:
should i remove it?

Reply:
I think hackautumn is fine. Scrappy reacts with aom by default